lib/pure/cookies.nim

```diff
@@ -8,11 +8,14 @@
 #
 
 ## This module implements helper procs for parsing Cookies.
+## https://tools.ietf.org/html/rfc6265
 
 import strtabs, times
 
 proc parseCookies*(s: string): StringTableRef =
-  ## parses cookies into a string table.
+  ## parses cookies into a string table. Example input: "a=1; foo=bar"
```

Comment:
"Set-Cookie" can contain "...; HttpOnly; Secure" attributes and they are not being handled by parseCookies. The proc is parsing key=value chunks only.